ospd/ospd.py

```diff
@@ -1707,9 +1713,20 @@ def create_scan(self, scan_id, targets, options, vts):
 
         @return: New scan's ID.
```

Comment:
The docstring should be updated for the already running case. It is returning None implicitly.